site/public/css/server.css

```diff
@@ -393,7 +393,17 @@ table tr.table-header {
     font-size: 20px;
 }
 
-#header-text>h2:nth-child(2) {
+#header-text>div {
+    display: block;
+    font-size: 1.5em;
+    margin-block-start: 0.83em;
+    margin-block-end: 0.83em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
```

Comment:
While I do think these tags are neat, they are marked [experimental](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-start) so I think we should look into some other tag for styling

Reply:
Those were the tags that Chrome was using to style the text as `<h2>`, per Chrome's developer tools.  But I'll look into other styling.

Reply:
I think it looking slightly different is also fine if trying to get the css rules right is a bear.

Reply:
The `margin-block-start` and `margin-block-end` lines appear to be forcing the element to be vertically centered.  We can accomplish the same thing with:
```css
position: absolute;
top: 50%;
transform: translateY(-50%);
```